package-lock.json

```diff
@@ -0,0 +1,2795 @@
+{
```

Comment:
Was this file added by mistake? There shouldn't be a package-lock.json file in the root directory.